paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/resources/LocalResourceRepository.kt

```diff
@@ -32,4 +46,13 @@ abstract class LocalResourceRepository protected constructor(
     namespace: ResourceNamespace,
     type: ResourceType
   ): Collection<ResourceItem> = throw UnsupportedOperationException("Not implemented yet")
+
+  /**
+   * Package accessible version of [.getMap].
+   * Do not call outside of [MultiResourceRepository].
+   */
+  open fun getMapPackageAccessible(
+    namespace: ResourceNamespace,
+    type: ResourceType
+  ): ListMultimap<String, ResourceItem>? = getMap(namespace, type)
```

Comment:
```suggestion
```

not needed

Reply:
`getMapPackageAccessible` is used to in [MultiResourceRepository.getResources](https://github.com/cashapp/paparazzi/blob/9573caf8173df9170b90322c7adb3bf7b4fdfe44/paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/MultiResourceRepository.kt#L479). We can discuss tomorrow on which alternative function we could use

Reply:
Ah, makes sense.  Thanks!